reaktive/src/commonMain/kotlin/com/badoo/reaktive/disposable/DisposableScope.kt

```diff
@@ -0,0 +1,43 @@
+package com.badoo.reaktive.disposable
+
+import com.badoo.reaktive.completable.Completable
+import com.badoo.reaktive.maybe.Maybe
+import com.badoo.reaktive.observable.Observable
+import com.badoo.reaktive.single.Single
+
+interface DisposableScope : Disposable {
+
+    fun <T : Disposable> T.scope(): T
+
+    fun cancel()
+
+    fun <T> Observable<T>.subscribeScoped(
```

Comment:
Can we do `Disposable.scoped()` instead of this? So I can do:
```kotlin
source.subscribe(...).scoped()
```

But it is easy to do mistake and use `scope()`, so we should rename it.